Add Enabling Istio Mesh as a prerequisite for serverless tutorial (#15324)
kyma-project/kyma: docs/03-tutorials/00-serverless/svls-11-set-asynchronous-connection-of-functions.md

- [Kyma CLI](https://github.com/kyma-project/cli)

> - Kyma installed locally or on a cluster

Reviewer: ```suggestion
- [Kyma installed](https://kyma-project.io/docs/kyma/latest/04-operation-guides/operations/02-install-kyma/) locally or on a cluster
```

> - [Istio sidecar injection enabled](../../04-operation-guides/operations/smsh-01-istio-enable-sidecar-injection.md) in the namespace where the functions will be deployed

Reviewer: ```suggestion
- [Istio sidecar injection enabled](../../04-operation-guides/operations/smsh-01-istio-enable-sidecar-injection.md) in the Namespace in which you want to deploy the Functions
```
